src/js/helpers/session_storage.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+var default_cookies = require('./cookies');
+
+module.exports = Object.assign({}, default_cookies, {
+
+  set: function(name, value, minutes, domain, excl_subdomains) {
+    // Don't break the build
+    excl_subdomains = minutes + domain + excl_subdomains;
+    sessionStorage.setItem(this.encodeData(name), value);
```

Comment:
I believe we don't need `this.encodeData` (and the corresponding `this.decodeData` in `get`):

```suggestion
    sessionStorage.setItem(name, value);
```